code/_onclick/hud/rendering/plane_master_group.dm

```diff
@@ -170,6 +172,22 @@
 /// If you wanna try someday feel free, but I can't manage it
 /datum/plane_master_group/popup
 
+/// This is janky as hell but since something changed with CENTER positioning after build 1609 we have to switch to the bandaid LEFT,TOP positioning
+/// using LEFT,TOP at or before 1609 will result in another broken offset for cameras
+#define MAX_CLIENT_BUILD_WITH_WORKING_SECONDARY_MAPS 1609
```

Comment:
Are you sure 1609 Is the minimum version? Like version below that all the way to 1600 don't have this problem

Reply:
Ok so in your PR body you said this

> However LEFT,TOP breaks the views for clients 1609 and below 

So you should make this 1600 or whatever version this bug starts to occur

Reply:
to my best understanding, 1609 is last time CENTER worked before it broke on secondary map screens

Reply:
> to my best understanding,

Sure,  but I would request you to check the byond release notes and double check just to be sure

Reply:
it's a good thing you pushed me to check because it's actually 515.1615 that breaks the secondary map positioning. So 515.1614 is the last working CENTER version